client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -140,7 +140,13 @@ export const QuickLinks = ( {
 				href="https://wp.me/logo-maker"
 				onClick={ trackDesignLogoAction }
 				target="_blank"
-				label={ translate( 'Create a logo' ) }
+				label={
```

Comment:
Oh wow. I spent a good ten minutes trying to understand this because I didn't realize the precedence was 

```
(getLocaleSlug() === 'en' ||	
	getLocaleSlug() === 'en-gb' ||		
	i18nCalypso.hasTranslation( 'Create a logo with Fiverr' )	)			
	? translate( 'Create a logo with Fiverr' ): translate( 'Create a logo' )
```

I was reading it left to right with the expression after the second `||`  as an entity.  🤦‍♀️ 

```
getLocaleSlug() === 'en' ||	
getLocaleSlug() === 'en-gb' ||		
( i18nCalypso.hasTranslation( 'Create a logo with Fiverr' )			
	? translate( 'Create a logo with Fiverr' )	: translate( 'Create a logo' ) )
```

I learn!

Reply:
Maybe jamming it all on one line wasn't the best idea, my bad. I was thinking how I wanted all the temporary code that will eventually be removed to be in one spot. But I should have created some intermediate variable.